src/Controllers/FilesController.js

```diff
@@ -82,6 +82,19 @@ export class FilesController extends AdaptableController {
   expectedAdapterType() {
     return FilesAdapter;
   }
+
+
+    /**
+    * Stream video file by serving data in chunks if FilesAdapter is GridStoreAdapter.
+    * If not; handle the request as usual with "getFileData".
+    */
+  handleVideoStream(filename, range, res, contentType) {
+      if (this.adapter.constructor.name == 'GridStoreAdapter') {
```

Comment:
Can you expose just check if the function exists on the adapter for forward compatibility?